### PR TITLE
Rename category header

### PR DESCRIPTION
**Pull request by medidatakerry**
This PR addresses issue #12

### CODE REVIEW

**Comment by medidatakerry** [comment]
This PR's merge is dependent on PR #16

File: MORK.podspec [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+  s.source          =   { :git => "https://github.com/mdsol/MORK.git", :tag => s.version.to_s }
+  s.source_files    =   'MORK/MORK.h',
+                        'MORK/ORKTaskResult+MORK.{h,m}',
+                        'MORK/ORKQuestionResult+MORK.{h,m}'
+
+  s.ios.deployment_target   = '8.0'
+  s.platform                = :ios, '8.0'
+  s.requires_arc            = true
+
+  s.dependency          'ResearchKit'
```
Comment (medidatakerry) [inline MORK.podspec]: I also added in some additional metadata for this to make our CocoaPod easier to find; obviously, we'll need to remember to push this (trunk?) once all's well.

File: MORK.podspec [hunk 2]
```diff
@@ -1,6 +1,18 @@
 Pod::Spec.new do |s|
-  s.name             =  'MORK'
-  s.version          =  '0.0.1'
-s.source_files     = 'MORK/MORK.h', 'MORK/ORKCollectionResult+MORK.m', 'MORK/ORKTaskResult+MORK.h', 'MORK/ORKQuestionResult+MORK.{h,m}'
-  s.dependency       'ResearchKit'
+  s.name            =   'MORK'
+  s.version         =   '0.0.1'
+  s.summary         =   "Extends Apple's ResearchKit to integrate with the Medidata Patient Cloud Gateway"
+  s.homepage        =   'https://github.com/mdsol/MORK'
+  s.source          =   { :git => "https://github.com/mdsol/MORK.git", :tag => s.version.to_s }
```
Comment (ncarroll-mdsol) [inline MORK.podspec]: Nitpick: newer ruby syntax `{git: 'http...', tag: s.version.to_s}`

**Comment by medidatakerry** [comment]
Updated!

**Pull request merged**